pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -62,11 +63,21 @@ func (r *reconciler) ReconcileCodeModuleCommunicationHosts(ctx context.Context,
 
 	oneAgentCommunicationHosts := oaconnectioninfo.GetCommunicationHosts(dynakube)
 
+	const conditionComponent = "CodeModule"
```

Comment:
couldn't we reuse the `OneAgentComponent` const here? 
I know that technically `CodeModule` is more accurate

but maybe we could change the `OneAgentComponent` to be more accurate
- Only do this if it does not break any naming

otherwise we could still just reuse `OneAgentComponent` and give it the old `strings.ToTitle()` to make it more "presentable" 😅

Reply:
I actually did that. Turns out `toTitle(..)` uppercases all letters, not just the initial one. :(

Reusing the existing constants would either mean to 
1. have the component part all lowercase, 
2. or all uppercase (using `toTitle(...)`)
3. or do some string-voodoo to just uppercase the initial rune.
4. 
possibilities 1 and 2 seemed bad in respect to readability, 3rd one seemed a bit over-engineered so the simplest solution I could think of was to use a constant... (changing the existing one seemed the worse idea as it would break exisiting behaviour)

Reply:
Did a bit of research, and yes the `ToTitle()` func is confusing AF.
There is a `.Title()` func that does kinda what I want. (makes only the first char in a word upper case) 
But even then, we would end up with the `Codemodule` or `Oneagent`

 so lets leave it as is, and do some refactoring incase in the future it will confuse us